lib/tasks/db.rake

```diff
@@ -8,21 +8,36 @@ namespace :db do
   desc "Will update design document with what is on db/designs/*.coffee"
   task design: :environment do
     design_dir = Rails.root.join 'db', 'designs'
-    files = Dir.glob File.join design_dir, '**', '*.coffee'
```

Comment:
for my own reference, this could be

    files = Dir[File.expand_path(File.join design_dir, '**', "*.{coffee,js}")]